ESQL sinh() tests failing on JDK 26

Several ESQL tests that verify `sinh()` behavior are failing on JDK 26 due to increased precision. Java 26 releases in March and we intend to upgrade the bundled JDK at that time. We'll need to update these tests to be compatible with Java 26 runtimes, but still support older ones as well.

### CI Link

https://gradle-enterprise.elastic.co/s/t7k3ovmrwtcz2/tests/task/:x-pack:plugin:esql:test/details/org.elasticsearch.xpack.esql.CsvTests/test%20%7Bcsv-spec:ints.Sinh%7D?top-execution=1

### Repro line

./gradlew ":x-pack:plugin:esql:test" --tests "org.elasticsearch.xpack.esql.CsvTests" -Dtests.method="test {csv-spec:ints.Sinh}" -Dtests.seed=71D90906F8562563 -Dtests.locale=vun-Latn-TZ -Dtests.timezone=Asia/Jerusalem -Druntime.java=26-pre -Druntime.java.build=32

### Does it reproduce?

Yes

### Applicable branches

main

### Failure history

_No response_

### Failure excerpt

```
    java.lang.AssertionError: 
    Data mismatch:
    row 0 column 1:a list containing
    row 0 column 1:0: expected <3.626860407847019> but was <3.6268604078470186>
    Actual:
    a:integer | sinh:double
    2         | 3.6268604078470186

    Expected:
    a:integer | sinh:double
    2         | 3.626860407847019
```

--- a/x-pack/plugin/esql/qa/testFixtures/src/main/resources/floats.csv-spec
+++ b/x-pack/plugin/esql/qa/testFixtures/src/main/resources/floats.csv-spec
@@ -336,11 +336,12 @@ sinh
 ROW a=1.8 
 | EVAL sinh=SINH(a)
 // end::sinh[]
+| EVAL sinh=round(sinh, 10)
 ;
 
 // tag::sinh-result[]
 a:double | sinh:double
-     1.8 | 2.94217428809568
+     1.8 | 2.9421742881
 // end::sinh-result[]
 ;
 

--- a/x-pack/plugin/esql/qa/testFixtures/src/main/resources/ints.csv-spec
+++ b/x-pack/plugin/esql/qa/testFixtures/src/main/resources/ints.csv-spec
@@ -671,10 +671,10 @@ a:integer | sin:double
 ;
 
 sinh
-ROW a=2 | EVAL sinh=SINH(a);
+ROW a=2 | EVAL sinh=round(SINH(a), 10);
 
 a:integer | sinh:double
-        2 | 3.626860407847019
+        2 | 3.6268604078
 ;
 
 asin

